manifest.yaml

```diff
@@ -259,6 +259,32 @@ packages:
  # https://github.com/openshift/machine-config-operator/pull/2421
  - conntrack-tools
 
+repo-packages:
+  # we always want the kernel from BaseOS
+  - repo: rhel-8-baseos
+    packages:
+      - kernel
+  # we want the one shipping in RHEL, not the equivalently versioned one in RHAOS
+  - repo: rhel-8-appstream
+    packages:
+      - nss-altfiles
+  - repo: rhel-8-server-ose
+    packages:
+      # eventually, we want the one from the container-tools module, but we're
+      # not there yet
+      - toolbox
+      # we want these from RHAOS to make sure they're in sync
+      # https://github.com/openshift/installer/pull/3062#issuecomment-582614009
+      - cri-o
+      - conmon
```

Comment:
Self-note for when we're ready for this: we can drop this now as per commit message in https://url.corp.redhat.com/8ef7f78.